docs/jwst/pipeline/main.rst

```diff
@@ -229,15 +204,94 @@ Control (WFS&C) use a dedicated processing flow through the pipeline stages.
    uses the :ref:`calwebb_wfs-image3 <calwebb_wfs-image3>` pipeline
    for stage 3 processing. This pipeline consists of the single step
    :ref:`wfs_combine <wfs_combine_step>`.
+
+Configuration File Deprecation
+==============================
+
+Up to version 1.1.1, the primary way specific pipelines were referred to was by
+their configuration file name, i.e. ``calwebb_detector1.cfg``. These
+configuration files were delivered as part of the JWST calibration package.
+Below is the table that matched configuration file to observing mode it was
+intended to be used with.
+
+Post-1.1.1, configuration files are no longer the primary identifier of
+pipelines. Instead, pipelines are identified by their full class name, i.e.
+``jwst.pipeline.Detector1Pipeline``, or by their simple name, or alias, i.e.
+``calwebb_detector1``. How a pipeline is run is determined by the input data and
+what step parameter reference file in CRDS is selected by that data. The reftype
+for each pipeline, or step, is determined by appending the class name of the
+step to the string ``pars-``. For example, the reftype for
+``jwst.pipeline.Detector1Pipeline`` is ``pars-detector1pipeline``. Which
+specific reference file for a reftype is then determined by the data, just as
+with any other reference file.
+
+As a result, there are a few pipelines that no longer exist explicitly by name,
+since they were only a configuration file for an already existing pipeline
+class. The pipelines continue to operate correctly for the specific cases,
+because the step parameter references pulled from CRDS will have the correct
+configuration. The following table lists the deprecated configuration files and
+what pipeline should now be referred to.
+
++---------------------------+-----------------------------------+-------------------+
+| Deprecated CFG            | Pipeline Class                    | Alias             |
++===========================+===================================+===================+
+| calwebb_nrslamp-spec2.cfg | `jwst.pipeline.Spec2Pipeline`     | calwebb_spec2     |
++---------------------------+-----------------------------------+-------------------+
+| calwebb_tso1.cfg          | `jwst.pipeline.Detector1Pipeline` | calwebb_detector1 |
++---------------------------+-----------------------------------+-------------------+
+| calwebb_tso-image2.cfg    | `jwst.pipeline.Image2Pipeline`    | calwebb_image2    |
++---------------------------+-----------------------------------+-------------------+
+| calwebb_tso-spec2.cfg     | `jwst.pipeline.Spec2Pipeline`     | calwebb_spec2     |
++---------------------------+-----------------------------------+-------------------+
+| calwebb_wfs-image2.cfg    | `jwst.pipeline.Image2Pipeline`    | calwebb_image2    |
++---------------------------+-----------------------------------+-------------------+
+
+The deprecated, up to version 1.1.1, configuration to mode mapping. This table is given only
```

Comment:
Perhaps move the "up to version ..." to the end of the sentence, so that the thought doesn't get broken up.